test/app/model/ChessEngine.spec.ts

```diff
@@ -1,11 +1,10 @@
-import { ChessEngine } from '../../../src/app/model/ChessEngine';
-import { Side, Square, SquareWithPiece } from '../../../src/app/model/Types';
-import { Chessboard } from '../../../src/app/model/Chessboard';
-import { Pawn } from '../../../src/app/model/pieces/Pawn';
-import { Player } from '../../../src/app/model/Player';
+import { ChessEngine } from '../../../src/app/model';
+import { Side, Square, SquareWithPiece } from '../../../src/app/model';
+import { Chessboard } from '../../../src/app/model';
+import { Pawn } from '../../../src/app/model';
 import 'jest-extended';
-import { Queen } from '../../../src/app/model/pieces/Queen';
-import { Knight } from '../../../src/app/model/pieces/Knight';
+import { Queen } from '../../../src/app/model';
+import { Knight } from '../../../src/app/model';
```

Comment:
Czy zmieniając te importy nie doprowadzasz ponownie to tych dziwnych błędów w testach i zawijaniu się JS w sobie?

Reply:
Testy sa OK - wystarczy zeranc na check na koncu PRa.

Reply:
hmmm... dobre pytanie, ale skoro testy działają to chyba nie ma to znaczenia... @nowakprojects co myślisz?

InteliJ podpowiedział, że mogę skrócić to dałem, bo i tak grzebałem w tym pliku - jest jeszcze chyba kilka innych miejsc, gdzie też mówi, że można skrócić